src/features/defi/providers/osmosis/components/OsmosisManager/Lp/Deposit/components/Deposit.tsx

```diff
@@ -174,6 +174,7 @@ export const Deposit: React.FC<DepositProps> = ({
 
       const inputAssetAmountBaseUnit = bnOrZero(inputAssetAmountPrecision)
         .multipliedBy(bn(10).pow(bnOrZero(inputAsset.precision)))
+        .toFixed(0, BigNumber.ROUND_DOWN)
         .toString()
```

Comment:
```suggestion
```
`toFixed()` [returns a string](https://mikemcl.github.io/bignumber.js/#toFix), so we don't actually need this.